### PR TITLE
Feature 2.1 rollpair

### PR DESCRIPTION
**Pull request by maxwong**
1. Supports rocksdb.

### CODE REVIEW

File: python/eggroll/core/pair_store/rocksdb.py [hunk 1]
```diff
@@ -12,47 +12,77 @@
...
 from eggroll.utils.log_utils import get_logger
+from eggroll.roll_pair.utils.pair_utils import get_data_dir
 
 L = get_logger()
 
-
 class RocksdbAdapter(PairAdapter):
-    env_lock = threading.Lock()
-    env_dict = dict()
+    db_lock = threading.Lock()
```
Comment (drougon) [inline python/eggroll/core/pair_store/rocksdb.py:33]: remove it

File: python/eggroll/core/pair_store/rocksdb.py [hunk 2]
```diff
@@ -12,47 +12,77 @@
...
+    db_dict = dict()
     count_dict = dict()
+    lock_dict = defaultdict(threading.Lock)
```
Comment (drougon) [inline python/eggroll/core/pair_store/rocksdb.py:36]: Remove from lock_dict when close

File: python/eggroll/core/pair_store/rocksdb.py (hunk 1 above)
Comment (maxwong) [reply python/eggroll/core/pair_store/rocksdb.py]: updated and removed

File: python/eggroll/core/pair_store/rocksdb.py (hunk 2 above)
Comment (maxwong) [reply python/eggroll/core/pair_store/rocksdb.py:35]: updated

**Pull request merged**